definitions/pipelines/somatic_exome.cwl

```diff
@@ -210,6 +210,7 @@ inputs:
     variants_to_table_fields:
         type: string[]
         default: [CHROM,POS,ID,REF,ALT,set,AC,AF]
+        doct: "The name of a standard VCF field or an INFO field to include in the output table"
```

Comment:
Looks like a typo snuck through:
```suggestion
        doc: "The name of a standard VCF field or an INFO field to include in the output table"
```

I might be inclined to say `The names of one or more standard VCF fields or INFO fields ...` since this is an array input.